### ISSUE TITLE
Add Polish Alphabet support

### ISSUE THREAD

**Issue by rafaehlers** [gravityview/AZ-Filters#42]
```
add_filter( 'gravityview_az_entry_filter_localization', 'add_polish_gravityview_az_entry_filter_localization' );
function add_polish_gravityview_az_entry_filter_localization( $languages = array() ) {
  $languages['pl_PL'] = __( 'Polish', 'gravityview-az-filters' );
  return $languages;
}

add_filter( 'gravityview_alphabets', 'add_polish_alphabet' );
function add_polish_alphabet( $alphabet = array() ) {
  $alphabet['pl_PL'] = array('A','Ą','B','C','Ć','D','E','Ę','F','G','H','I','J','K','L','Ł','M','N','Ń','O','Ó','P','R','S','Ś','T','U','W','Y','Z','Ź','Ż');
  return $alphabet;
}
```

**Comment by zackkatz** [comment]
@rafaehlers This is a pretty straightforward next project for you.

**Comment by zackkatz** [comment]
I thought this was familiar; it's been in the `develop` branch for 3.5 years! Closed in a78cb01.
